app/Models/FormAuth.php

```diff
@@ -70,7 +70,7 @@ public static function deleteCookie(): void {
 			@unlink(DATA_PATH . '/tokens/' . $token . '.txt');
 		}
 
-		if (rand(0, 10) === 1) {
+		if (random_int(0, 10) === 1) {
```

Comment:
Same here, no need of the overhead of a cryptographically secure generator

Reply:
done